### DISCUSSION TITLE
Is there a way to exclude a topic from the search results or deprioritize matches in that topic?

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#486] [0. General bugs and questions] [answered]
I use GitBook to publish end user documentation.

The documentation includes topics for the product release notes, which date back over a few years.

We've had an issue where some searches turn up the release notes results first, even though that's probably the last match that users actually want when they're looking for help on how to do something. In some cases, there are several release note matches before the user sees any matches from the actual help content.

Is there a way to indicate that a search should ignore a topic, or deprioritize the topic so that matches are at the bottom of the list?

**Comment by SamyPesse** [comment]
Not at the moment. But this is an interesting and relevant use case that we need to find a way to address. I think we need to find a way to tackle it without having GitBook users manually control search at the page level; for example, we could use the modification date of the page to boost/unboost the ranking of the pages

**Comment by JaniceManwiller** [comment]
Not sure if modification date would completely cover it, since we add new release notes every week. But thank you for responding and for thinking about it.

If worst comes to worst we might consider moving the release notes to a different space.

**Reply by aikoii** [reply, marked as answer]
Hey @JaniceManwiller, we just shipped a feature that lets you control which pages are indexed in search. This should help you prioritize your help content over release notes 🤞. You can find more details in #730

**Reply by JaniceManwiller** [reply]
That's great to hear - and I know others also wanted that option - although we are in any case about to move the release notes out of the guide in favor of automatically generated content on our website.

**Comment by Alf-Melmac** [comment]
You can now exclude selected pages from the search index: https://github.com/orgs/GitbookIO/discussions/730

**Reply by pbambynek** [reply]
Do you plan to add a way to assign priority levels (e.g., low, medium, high) or a numerical ranking value to documents that would influence their position in search results? Now you allow only exclude now but this feature would be only final sort before presentation.

For example:

If we have a very important breaking change in the changelog, we would like it to appear at the top of the list when users search.

Conversely, we might want certain documents to always appear at the end of the list, even if they match the query 
e.g.:
API-documentation: high priority;
news: medium (default);
changelog: low priority;

This would give us much finer control over how search recommendations are ordered and ensure that the most critical documentation is surfaced first and not obscured by latest cosmetic change to not important docs.

Thank you for considering this improvement!
